test/integration/commands/redactCommandTest.ts

```diff
@@ -446,4 +446,56 @@ describe("Test: The redaction command - if not admin", function () {
         assert.equal(rooms.length, 1);
         assert.equal(rooms[0], targetRoom);
     });
+
```

Comment:
Pretty sure this is failing because it is attempting to quarantine - which is an admin API call - when the bot is not an admin. Moving this up to the `"Test: The redaction command - if admin"` test staring on line 9 should fix this I think.